lib/srv/usermgmt.go

```diff
@@ -393,6 +431,7 @@ func (u *HostUserManagement) createUser(name string, ui services.HostUsersInfo)
 			return trace.Wrap(err)
 		}
 
+		log.InfoContext(context.Background(), "Creating host user", "gid", userOpts.GID)
```

Comment:
It adds a second gid to the attributes which is not what I initially expected 😅 It makes some sense though since appending attrs to the logger is probably the simplest possible implementation and that's more or less what I would expect from the stdlib. It also looks like we can influence that behavior if we really want to, but for now I've fixed this by dropping it from the scoped logger.